### PR TITLE
Add parameter skipSDKGeneration

### PR DESCRIPTION
**Pull request by pshao25**

### CODE REVIEW

File: src/CADL.Extension/Emitter.Csharp/src/emitter.ts [hunk 1]
```diff
@@ -71,11 +71,13 @@ import { exec } from "child_process";
 export interface NetEmitterOptions {
     outputFile: string;
     logFile: string;
+    runCodegen: 'true' | 'false';
```
Comment (chunyu3) [inline src/CADL.Extension/Emitter.Csharp/src/emitter.ts:74]: Can it be boolean?

File: src/CADL.Extension/Emitter.Csharp/src/emitter.ts [hunk 2]
```diff
@@ -71,11 +71,13 @@ import { exec } from "child_process";
...
 }
 
 const defaultOptions = {
     outputFile: "cadl.json",
-    logFile: "log.json"
+    logFile: "log.json",
+    runCodegen: "true"
```
Comment (lirenhe) [inline src/CADL.Extension/Emitter.Csharp/src/emitter.ts:80]: Can we find a better name to describe it, 'GenerateSDK'?

File: src/CADL.Extension/Emitter.Csharp/src/emitter.ts (hunk 1 above)
Comment (pshao25) [reply src/CADL.Extension/Emitter.Csharp/src/emitter.ts:74]: No. Compiler takes it as string even you declare it as boolean.

File: src/CADL.Extension/Emitter.Csharp/src/emitter.ts (hunk 2 above)
Comment (ArcturusZhang) [reply src/CADL.Extension/Emitter.Csharp/src/emitter.ts:80]: I personally think that a boolean option with a `true` as default is not very intuitive. Could we flip this switch to something like `skipCodegen` or `skipSDKGeneration`?

**Review by lirenhe — APPROVED** [review]

**Pull request merged**